src/ResourceManager/Resources/Commands.Resources.Test/packages.config

```diff
@@ -20,7 +20,7 @@
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.5" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.6" targetFramework="net452" />
```

Comment:
Because I installed the authorization package use command in 'Package Manager Console'  So changed the version of 'Microsoft.Rest.ClientRuntime' to 2.3.6 automatically.